### PR TITLE
Create event form for catalog

### LINKED ISSUE #4: Create Event Form for Catalog
## Instructions
Events are a core component of AISS’s event management system. You are going to jumpstart this project by making a form to create an event. For this task you’re going to…
- Set-up the basic skeleton for a form to create new events in the catalog using Chakra UI components
- Add validation using Yup

## Notes
- Refer to DB Model for which fields to include
  - For any fields with `ENUM` values, use dropdowns
  - For any fields marked `NOT NULL`, make the fields required
- Refer to PNP code example for how to implement Yup into a form
- The design for the event creation form hasn’t been finalized yet, so don’t worry too much about it yet
- Make your best approximation for validation

## Acceptance Criteria
- Place the form in a new file located in `src/components/CreateEventForm/CreateEventForm.jsx`
- All fields of the Catalog table in the DB model should be included
- Input validation should exist for all fields as needed
- The user cannot submit the form if violating any input validation
After submitting the form, the form data should be console logged (we’ll send it to the backend later)

## Resources
- [Chakra Documentation](https://chakra-ui.com/)
- [Yup Documentation](https://github.com/jquense/yup)
- [PNP Form Example w/ Yup Validation](https://github.com/ctc-uci/patriots-and-paws-frontend/pull/15/files#diff-f9d37c71ec6abb50162961b81a9306566fcd8f84a175269035ee3d908f62c002)
- [DB Model](https://www.figma.com/file/NDPdo1MR13qgsB4LubtzyZ/Database-Model?type=whiteboard&node-id=0-1&t=m1J0vcLI2mzIb5SN-0)

#### As always, feel free to message Michael _and_ Michelle if you have any questions!

### PR DESCRIPTION
**Pull request by ctc-devops**
Authors:
 ### What does this PR contain?
 ### How did you test these changes?
 ### Attach images (if applicable)
 Closes #4

### CODE REVIEW

File: src/components/CreateEventForm/CreateEventForm.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    host: yup.string().required('Host required').max(50, "Host exceeds 50 character limit"),
+    title: yup.string().required('Title required').max(50, "Title exceeds 50 character limit"),
+    event_type: yup.string().required('Email required'),
+    subject: yup.string().required('Subject required'),
+    description: yup.string().required('Description required').max(50, "Description exceeds 50 character limit"),
+    year: yup.string().required('Year required'),
+});
+
+
+function CreateEventForm() {
```
Comment (michellelin1) [inline src/components/CreateEventForm/CreateEventForm.jsx]: pls change this to an arrow function as it is better practice and to be consistent with the rest of the codebase

File: src/components/CreateEventForm/CreateEventForm.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      formState: { errors },
+    } = useForm({
+      resolver: yupResolver(schema),
+    });
+
+
+    return (
+      <div className={styles['form-padding']}>
+        {/* eslint-disable-next-line no-console */}
+        {/* <form onSubmit={handleSubmit((data) => console.log(data))}> */}
```
Comment (michellelin1) [inline src/components/CreateEventForm/CreateEventForm.jsx]: pls remove the comments on line 31 and 42

File: src/components/CreateEventForm/CreateEventForm.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    //   control,
+      handleSubmit,
+      formState: { errors },
+    } = useForm({
+      resolver: yupResolver(schema),
+    });
+
+
+    return (
+      <div className={styles['form-padding']}>
```
Comment (michellelin1) [inline src/components/CreateEventForm/CreateEventForm.jsx]: pls use change div to chakra Box and if the css is very minimal and only used once, use chakra's in-line styling

**Review by michellelin1 — DISMISSED** [review]
great job, everything works!! we just have a few minor changes so once those are fixed we'll approve this pr :)

**Review by michellelin1 — COMMENTED** [review]
also make sure to push the `package.json` so others can install the dependencies when they run this

**Review by ThatMegamind — APPROVED** [review]
Looks good to us now! (please excuse the extra commits we were fixing some package.json issues)

**Pull request merged**